chart/templates/proxy-restarter-role.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+# Licensed under the MIT License. See License-MIT.txt in the project root for license information.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: proxy-restarter
+  labels:
+    app: {{ template "gitpod.fullname" . }}
+    component: proxy-restarter
+    kind: role
+    stage: {{ .Values.installation.stage }}
+rules:
+  - apiGroups: ['apps']
+    resources: ['deployments']
+    verbs: ['get','patch']
+  - apiGroups: ['']
+    resources: ['pods']
+    verbs: ['get', 'list']
+  - apiGroups: ['']
```

Comment:
No, it is not needed and I should remove it. But I let it in there because if you do not agree to restart the deployment, I would already have the possibility to send a `kubectl exec -it <POD> -- nginx -s reload` to the `proxy` pods. ;)

Reply:
:+1:  A comment would be awesome

Reply:
I have removed it.